Formula/chuck.rb

```diff
@@ -13,6 +13,9 @@ class Chuck < Formula
 
   depends_on :xcode => :build
 
+  # patch used to include new OSX version in the version check
+  patch :DATA
```

Comment:
Can you please submit add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!

Additionally, although we still want this upstream, an `inreplace` with `MacOS.version` may be a more future-proof version.